CalculatorApp.Tests/AddMethodShould.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using CalculatorApp;
+using FluentAssertions;
+namespace CalculatorApp.Tests
+{
+    public class AddMethodShould
```

Comment:
test class name should be like this 
ClassUnderTestNameTest
in your case it would be : CalculatorTests